### DISCUSSION TITLE
Option to force GBA or HD texture?

### DISCUSSION THREAD

**Discussion by goldenyoshi22** [chrislo27/PolyrhythmMania#32] [Ideas]
An option (maybe in "VIDEO" tab) to either force the GBA or HD texture, or don't force a texture. (Don't force by default)

## Justification
The reason for this is because there are many slow computers like mine, and it lags with the HD texture.

## Potential Issues
If there are maps that use custom textures, it may look weird. You could maybe have an option to let custom textures override or not.

# 

I am typing this on mobile, so I can't make an image mockup. If I could, I would make a checkmark with the text "Force texture". When it's on, an arrow switch appears with the options, "GBA" and "HD".

**Comment by chrislo27** [comment]
This is a good suggestion. I think I will flesh it out into its own "Graphics" settings menu and add a dropdown for the various options you described.

This suggestion has been elevated to an issue, #33.
